docs/section10/10-13A.md

```diff
@@ -0,0 +1,35 @@
+# Airside systems – Rule 10-13A    
+**Schema Version:** 0.0.13    
+**Mandatory Rule:** True    
+**Rule ID:** 10-13A    
+ 
+**Rule Description:** For HVAC systems designed, mechanical heating equipment efficiencies shall be adjusted to remove the supply fan energy from the efficiency rating.  
+
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Table G3.1 Section 10 b.  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. HVAC with heat pump heating.  
+
+**Function Call:** None  
+
+**Manual Check:**  None  
+
+## Rule Logic:   
+**Applicability Check 1:**   
+- For each hvac system in the P_RMR: `for hvac_p in P_RMR...HeatingVentilationAirConditioningSystem:`
+    - For each heating system in the HVAC system: `for heating_system_p in hvac_p.heating_system:`
+        - Check that heating system type is equal to heat_pump:: `if heating_system_p.heating_system_type == "HEAT_PUMP":`
+            - Reset no_fan_eff_entered boolean variable: `no_fan_eff_entered = FALSE`
```

Comment:
similar comment as #371 , if no Heat Pump system is found, we could have a not applicable outcome for this rule. There could be a Case 3 which provides the not applicable outcome for RMD that doesn't have heat pumps

Reply:
@supriyagoel Based on previous decisions, we do not explicitly include the N/A as an outcome. It is my understanding that the RCT team will automatically do this for systems that are not applicable based on the applicability check description.